Documentation/guides/building-apps/build-properties.md

```diff
@@ -1303,6 +1303,15 @@ This is only used when building `system` applications.
 
 Support for this property was added in Xamarin.Android 11.3.
 
+## AndroidStripIL
```

Comment:
If others have a better idea on the name, basically you use this in combination with AOT and it will strip any IL that was AOT'd.

For best results, we'd turn off AndroidEnableProfiledAot by default -- so AndroidStripIL will AOT & Strip all the IL as a good default setting.

@fanyang-mono has some example `.apk` sizes from a `dotnet new android` project:

| AndroidStripIL | AndroidEnableProfiledAot | App size | 
| -- | -- | -- |
| false          | true                     | 7.7M     | 
| true           | true                     | 7.7M     | 
| false          | false                    | 8.4M     | 
| true           | false                    | 8.1M     |

Reply:
Ok looks like there is a `$(WasmStripILAfterAOT)`:

https://github.com/dotnet/runtime/blob/2ed506c010b41c41c01997c81df977831f0fa217/src/mono/wasm/build/WasmApp.targets#L88-L89

Reply:
$(AndroidStripILAfterAOT) seems logical to align?

Reply:
Changed parameter name to `AndroidStripILAfterAOT `